src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,215 @@
+package controller;
+
+import model.*;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+/**
+ * Created by Jef1771 on 7/15/17.
+ */
+public class Controller {
+
+    //DAO's
+    PlayerDAO playerDAO;
+    AreaDAO areaDAO;
+    EnemyDAO enemyDAO;
+    WeaponDAO weaponDAO;
+    TreasureDAO treasureDAO;
+
+
+    Map<Integer, Enemy> enemies;
+    Area area;
+    Player currentPlayer;
+
+    public Controller(){
+        playerDAO = new PlayerDAO();
+        areaDAO = new AreaDAO();
+        enemyDAO = new EnemyDAO();
+        weaponDAO = new WeaponDAO();
+        treasureDAO = new TreasureDAO();
+    }
+
+    public void setCurrentPlayer(String name){
+        currentPlayer = playerDAO.getPlayer(name);
+        area = currentPlayer.currentArea;
+        populateEnemies();
+    }
+
+    public void setCurrentArea(Area a){
+        area = a;
+        currentPlayer.currentArea = a;
+        populateEnemies();
+    }
+
+    public void createNewPlayer(String name){
+        currentPlayer = playerDAO.createPlayer(name);
+        area = currentPlayer.currentArea;
+        populateEnemies();
+    }
+
+    //VIEW USE THESE METHODS TO GET INFO YOU NEED
+
+    /**
+     * Get a list of all the player names in the DB
+     *
+     * @return List of all player names
+     */
+    public List<String> getAllPlayers(){
+        return playerDAO.getNamesOfAllPlayers();
+    }
+
+    /**
+     * Get a Map of all the enemies names in the current Area
+     *
+     *
+     *
+     * @return List of all enemies names
+     */
+    public Map<Integer, String> getAllEnemies(){
+        Map<Integer, String> enemyNames = new HashMap<>();
+        enemies.forEach((id, enemy) -> {
+            enemyNames.put(id, enemy.name);
+        });
+        return enemyNames;
+    }
+
+    /**
+     * Get the current player's name
+     *
+     * @return String of player name
+     */
+    public String getCurrentPlayerName(){
+        return currentPlayer.name;
+    }
```

Comment:
Can you return the player object here or add a method where I can get player current health?